flappy-dird.applescript

```diff
@@ -0,0 +1,40 @@
+property projectPath : ""
+
+set myAlias to path to me
+set myPath to POSIX path of myAlias -- Converts alias to a Unix style path
+
+-- Find the last occurrence of "/" (slash)
+set lastSlash to the offset of "/" in (reverse of characters of myPath) as string
+
+-- Remove the filename from the path
+set projectPath to text 1 thru -((lastSlash + 1)) of myPath & "/"
+
+property buf2Path : projectPath & "buf2"
+-- A subroutine to set Finder window's target
+on setFinderTarget(buf)
+    tell application "Finder" to set target of front Finder window to (projectPath & buf as POSIX file)
+    tell application "Finder" to tell front window to update every item
```

Comment:
FWIW this is a small semantic change - we didn't tell finder to update every item *except* at the absolute start of the game but now we're going to do it on every frame change.

I originally thought that this might cause problems (applescript is slow!) but I realize now that unless it takes enough time to prevent us from hitting our target framerate (unlikely) latency here doesn't matter because the user can still select a file here. Latency here is ~the same as just sleeping for longer at the end of the loop!